examples/demo-app/src/components/sharing/cloud-storage.js

```diff
@@ -157,12 +164,12 @@ const ExportCloudModal = ({
                 )}
                 {error && (
                   <div className="subtitle" style={{color: 'red', fontWeight: 500}}>
-                    {error.error}
+                    {providerName}: {error.error}
                   </div>
                 )}
                 {metaUrl && [
                   (<SharingUrl key={0} url={sharingLink} message={'Share your map with other users'}/>),
-                  (<a href={folderLink} target="_blank" rel="noopener noreferrer" style={{textDecoration: 'underline'}}>Go to your Kepler.gl Dropbox folder</a>)
+                  (folderLink && <a href={folderLink} target="_blank" rel="noopener noreferrer" style={{textDecoration: 'underline'}}>Go to your Kepler.gl {providerName} page</a>)
```

Comment:
Nice, providerName fits well here!

Reply:
With this 'old approach' (not our first one), the renderMeta method in carto-provider won't be used (delete or at least comment it out). But we might want to discuss about that in the future, as I think it is bit cleaner than this current approach.